pkgs/tools/wayland/cliphist/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "cliphist";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "sentriz";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-CZW7dhOd7E74VNjnvhxvSSUZQtbkGi4uRUM9YQCuJZw=";
+  };
+
+  vendorSha256 = "sha256-rxE4fNS3THWAfqIpKLPQ+a9dl1fDnIlMfvblb7JqfBc=";
+
+  nativeBuildInputs = [ installShellFiles ];
```

Comment:
That is not used anywhere.

Reply:
I have removed the nativeBuildInputs but seems like it does not get built if I remove the vendorSha256?

Reply:
Yeah you did it right, keep the vendorSha256.